cli/cmd/uninstall.go

```diff
@@ -36,9 +36,16 @@ var uninstallCmd = &cobra.Command{
 		}
 		ctx := cmd.Context()
 
+		errMessage := fmt.Errorf("Odigos is not installed in the cluster")
```

Comment:
The current message lacks context.

How about? 
```
Odigos is not currently installed in the cluster, so there is nothing to uninstall
```

Reply:
I have updated the error message with your suggested message